projects/1.16/assets/geolosys/geolosys/lang/zh_cn.json

```
@@ -1 +1,103 @@
+  "block.geolosys.gold_ore": "金",
+  "block.geolosys.lapis_ore": "青金石",
+  "block.geolosys.quartz_ore": "杂石英",
+  "block.geolosys.kimberlite_ore": "金伯利岩",
+  "block.geolosys.beryl_ore": "绿柱石",
+  "block.geolosys.ancient_debris_ore": "古代遗迹",
+  "block.geolosys.nether_gold_ore": "贫金",
+  "block.geolosys.lignite_ore_sample": "褐煤样品",
+  "block.geolosys.bituminous_coal_ore_sample": "烟煤样品",
+  "block.geolosys.anthracite_coal_ore_sample": "褐煤样品",
+  "block.geolosys.hematite_ore_sample": "赤铁矿样品",
+  "block.geolosys.limonite_ore_sample": "褐铁矿样品",
+  "block.geolosys.malachite_ore_sample": "孔雀石样品",
+  "block.geolosys.azurite_ore_sample": "蓝铜矿样品",
+  "block.geolosys.cassiterite_ore_sample": "锡石样品",
+  "block.geolosys.teallite_ore_sample": "硫锡铅矿样品",
+  "block.geolosys.galena_ore_sample": "方铅矿样品",
+  "block.geolosys.bauxite_ore_sample": "铝土矿样品",
+  "block.geolosys.autunite_ore_sample": "钙铀云母样品",
+  "block.geolosys.platinum_ore_sample": "铂样品",
+  "block.geolosys.sphalerite_ore_sample": "闪锌矿样品",
+  "block.geolosys.coal_ore_sample": "煤样品",
+  "block.geolosys.cinnabar_ore_sample": "朱砂样品",
+  "block.geolosys.gold_ore_sample": "金样品",
+  "block.geolosys.lapis_ore_sample": "青金石样品",
+  "block.geolosys.quartz_ore_sample": "杂石英样品",
+  "block.geolosys.kimberlite_ore_sample": "金伯利岩样品",
+  "block.geolosys.beryl_ore_sample": "绿柱石样品",
+  "block.geolosys.ancient_debris_ore_sample": "古代遗迹样品",
+  "block.geolosys.nether_gold_ore_sample": "贫金样品",
+  "item.geolosys.iron_cluster": "铁簇",
+  "item.geolosys.gold_cluster": "金簇",
+  "item.geolosys.copper_cluster": "铜簇",
+  "item.geolosys.tin_cluster": "锡簇",
+  "item.geolosys.silver_cluster": "银簇",
+  "item.geolosys.lead_cluster": "铅簇",
+  "item.geolosys.aluminum_cluster": "铝簇",
+  "item.geolosys.uranium_cluster": "铀簇",
+  "item.geolosys.nickel_cluster": "镍簇",
+  "item.geolosys.platinum_cluster": "铂簇",
+  "item.geolosys.zinc_cluster": "锌簇",
+  "item.geolosys.osmium_cluster": "锇簇",
+  "item.geolosys.yellorium_cluster": "黄铀簇",
+  "item.geolosys.ancient_debris_cluster": "古代遗迹聚集",
+  "item.geolosys.nether_gold_cluster": "贫金簇",
+  "item.geolosys.copper_ingot": "铜锭",
+  "item.geolosys.tin_ingot": "锡锭",
+  "item.geolosys.silver_ingot": "银锭",
+  "item.geolosys.lead_ingot": "铅锭",
+  "item.geolosys.aluminum_ingot": "铝锭",
+  "item.geolosys.nickel_ingot": "镍锭",
+  "item.geolosys.platinum_ingot": "铂锭",
+  "item.geolosys.zinc_ingot": "锌锭",
+  "item.geolosys.aluminum_nugget": "铝块",
+  "item.geolosys.copper_nugget": "铜块",
+  "item.geolosys.lead_nugget": "铅块",
+  "item.geolosys.nickel_nugget": "镍块",
+  "item.geolosys.platinum_nugget": "铂块",
+  "item.geolosys.silver_nugget": "银块",
+  "item.geolosys.tin_nugget": "锡块",
+  "item.geolosys.zinc_nugget": "锌块",
+  "item.geolosys.peat_coal": "泥炭",
+  "item.geolosys.lignite_coal": "褐煤",
+  "item.geolosys.bituminous_coal": "烟煤",
+  "item.geolosys.anthracite_coal": "无烟煤",
+  "item.geolosys.lignite_coal_coke": "褐煤焦炭",
+  "item.geolosys.bituminous_coal_coke": "烟煤焦炭",
+  "item.geolosys.field_manual": "现场手册",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "item.geolosys.field_manual": "野外探矿手册",
```
若无必要，译名需与1.12一致。